src/content.ts

```diff
@@ -1610,6 +1610,7 @@ export class RikaiContent {
       // Add the popup div
       const popup = doc.createElement('div');
       popup.setAttribute('id', 'rikaichamp-window');
+      popup.setAttribute('lang', 'ja-JP');
```

Comment:
Thank you! We actually do this for the kanji pop-up for some of the fields, but not all of them 🤷‍♀️.

I think we should use 'ja' however, as per https://www.w3.org/International/articles/language-tags/ :

> **The golden rule when creating language tags is to keep the tag as short as possible.** Avoid region, script or other subtags except where they add useful distinguishing information. For instance, use `ja` for Japanese and not `ja-JP`, unless there is a particular reason that you need to say that this is Japanese _as spoken in Japan, rather than elsewhere_.